packages/dd-trace/src/telemetry/send-data.js

```diff
@@ -54,8 +58,28 @@ function sendData (config, application, host, reqType, payload = {}) {
     host
   })
 
-  request(data, options, () => {
-    // ignore errors
+  request(data, options, (error) => {
+    if (error && process.env.DD_API_KEY) {
+      if (agentTelemetry) {
+        log.info('Agent telemetry failed, started agentless telemetry')
+        agentTelemetry = false
+      }
+      const backendHeader = { ...options.headers, 'DD-API-KEY': process.env.DD_API_KEY }
+      const backendOptions = {
+        ...options,
+        url: 'https://all-http-intake.logs.datad0g.com/api/v2/apmtelemetry',
```

Comment:
Is `datad0g.com` the real destination hostname? It looks like a value we would use for testing.

Reply:
Instrumentation telemetry team gave me that URL. @yshapiro-57 do you want to verify?

Reply:
I searched through some codebases and it looks like you're right!

Reply:
I believe you should use the endpoints [from this document](https://github.com/DataDog/instrumentation-telemetry-api-docs/blob/main/GeneratedDocumentation/ApiDocs/v2/development.md#agent)